src/lib/pke/serialization.cpp

```diff
@@ -110,6 +143,29 @@ void bind_serialization(pybind11::module &m) {
           py::arg("filename"), py::arg("obj"), py::arg("sertype"));
     m.def("DeserializeEvalKey", static_cast<std::tuple<EvalKey<DCRTPoly>,bool> (*)(const std::string&, const SerType::SERJSON&)>(&DeserializeFromFileWrapper<EvalKey<DCRTPoly>, SerType::SERJSON>),
           py::arg("filename"), py::arg("sertype"));
+
+    // JSON Serialization to string
+    m.def("Serialize", &SerializeToStringWrapper<CryptoContext<DCRTPoly>, SerType::SERJSON>,
+          py::arg("obj"), py::arg("sertype"));
+    m.def("DeserializeCryptoContextString", &DeserializeFromStringWrapper<CryptoContext<DCRTPoly>, SerType::SERJSON>,
```

Comment:
To stay closer to the `openfhe-development` code base,  this should be renamed to `DeserializeCryptoContext` and the existing one should become `DeserializeCryptoContextFromFile`.

This would kill backward compatibility, but according to [semver](https://semver.org/#spec-item-4), this would be OK since we are on version `0.y.z`.

(repeat remark 4x times for the other functions)

Reply:
I see now you already remarked this in your original message. @yspolyakov, can you weigh in on this?